packages/embed/src/factories/create-popover/create-popover.ts

```diff
@@ -97,6 +97,47 @@ const buildTooltip = (message: string, hide: () => void) => {
   return container
 }
 
+const LOCAL_STORAGE_NOTIFICATION_DATA = 'tfNotificationData'
+
+type LocalStorageCUIData = {
+  formId: string
+  expireDate: number
+}
+const today = new Date().getTime()
+
+const cuiLocaslStorageData = localStorage.getItem(LOCAL_STORAGE_NOTIFICATION_DATA)
+
+const getLocalStorageData = () => cuiLocaslStorageData && JSON.parse(cuiLocaslStorageData)
+
+function saveToLocalStorage(data: LocalStorageCUIData) {
+  if (!data) return null
+  localStorage.setItem(LOCAL_STORAGE_NOTIFICATION_DATA, JSON.stringify(data))
```

Comment:
For not data the method returns `null` but otherwise `undefined`. I would make it consistent:
```suggestion
  data && localStorage.setItem(LOCAL_STORAGE_NOTIFICATION_DATA, JSON.stringify(data))
```